coderd/rbac/roles_test.go

```diff
@@ -354,6 +354,17 @@ func TestRolePermissions(t *testing.T) {
 				false: {setOtherOrg, setOrgNotMe, templateAdmin, userAdmin},
 			},
 		},
+		{
+			Name: "InboxNotification",
+			Actions: []policy.Action{
+				policy.ActionCreate, policy.ActionRead, policy.ActionUpdate,
+			},
+			Resource: rbac.ResourceInboxNotification.WithID(uuid.New()).InOrg(orgID).WithOwner(currentUser.String()),
+			AuthorizeMap: map[bool][]hasAuthSubjects{
+				true:  {owner, orgMemberMe, orgAdmin},
```

Comment:
Why does `orgMemberMe` have privileges to `policy.ActionCreate` inbox notifications?

Reply:
From my RBAC understanding `orgMemberMe` is the current user. 
Associated [to this part](https://github.com/coder/coder/blob/main/coderd/rbac/roles_test.go#L113C69-L113C80) on the which we use the `currentUser` id.

In the definition of the ResourceInboxNotification, I assign the `currentUser` to `InboxNotification` using the `WithOwner(currentUser.String())` method.

TL;DR - `orgMemberMe` is the owner of the resource in the test - and the owner of the resource should be able to access it.

Reply:
I think we might want `memberMe` in here as well?

Reply:
@defelmnq  I understand why the current user should be able to _read_ their own messages, but why should they be allowed to _create_ new messages? That should be a system privilege.